src/classes/PS_CommitmentRequest.cls

```diff
@@ -330,6 +345,12 @@ public inherited sharing class PS_CommitmentRequest {
         return accounts.isEmpty() ? null : accounts[0];
     }
 
+    public with sharing class AchData {
```

Comment:
I'd like to combine this with the Gift Entry version of AchData eventually. I think there's an opportunity for having a class whose sole purpose is containing the classes that represent the messages we send to Elevate.